.github/workflows/ci.yml

```diff
@@ -55,13 +55,13 @@ jobs:
 
       - uses: golangci/golangci-lint-action@v3
         with:
-          version: v1.50.1
+          version: v1.51.2
```

Comment:
Once you rebase then this will be the only change left.

Reply:
I also upgrade to this version of glonagci-lint now along with resolving the code issues.